uber/site_sections/registration.py

```diff
@@ -81,7 +81,7 @@ def index(self, session, message='', page='0', search_text='', uploaded_id='', o
         }
 
     @log_pageview
```

Comment:
We have an "Omit badge number" checkbox on the at-the-door admin reg form, which is used when someone wants to enter a person without automatically assigning them a badge and checking them in.

However, since MAGStock doesn't do numbered, badges, it makes sense to instead have a "Check this person in" box, so that when using the admin at-the-door reg form, the admin can say whether they're just entering someone who will be checked in later, or entering someone who is already present and should automatically be checked in when the form is submitted.